src/main/java/io/aiven/kafka/connect/transforms/ExtractTimestampConfig.java

```diff
@@ -40,10 +47,25 @@ static ConfigDef config() {
                 ConfigDef.NO_DEFAULT_VALUE,
                 new ConfigDef.NonEmptyString(),
                 ConfigDef.Importance.HIGH,
-                FIELD_NAME_DOC);
+                FIELD_NAME_DOC)
+            .define(
+                EPOCH_RESOLUTION_CONFIG,
+                ConfigDef.Type.STRING,
+                "ms",
+                new ConfigDef.NonEmptyString(),
```

Comment:
we should specify the valid values here with ConfigDef.ValidString.in(...)

Reply:
True will exclude it so far.